src/utils/traceability.util.ts

```diff
@@ -0,0 +1,31 @@
+import {ReportUtility} from "./report.util";
+
+export class TraceabilityUtility {
+    private static coveredSpecs: Set<string> = new Set<string>();
+
+    public static registerCoveredSpecs(specList: string[]): void {
+        this.addCoveredSpecs(specList);
+        this.dumpCoveredSpecs();
+        this.clearCoveredSpecs();
+    }
+
+    public static addCoveredSpecs(specsList: string[]): void {
+        specsList.forEach(spec => this.coveredSpecs.add(spec));
+    }
+
+    public static clearCoveredSpecs(): void {
+        this.coveredSpecs.clear();
+    }
+
+    public static dumpCoveredSpecs(): void {
```

Comment:
This method and the getCoveredSpecsPrettyString() look interesting for their use. Would it be worth to add them into TRACEABILITY.md?

Reply:
getCoveredSpecsPrettyString() is just for internal use inside the library (used by test-case.reporter.js).

dumpCovereSpecs() and registerCoveredScpecs are already docummented in the *.md file